sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -106,27 +106,27 @@ trait InvokeLike extends Expression with NonSQLExpression {
 }
 
 /**
- * Invokes a static function, returning the result.  By default, any of the arguments being null
- * will result in returning null instead of calling the function.
- *
- * @param staticObject The target of the static call.  This can either be the object itself
- *                     (methods defined on scala objects), or the class object
- *                     (static methods defined in java).
- * @param dataType The expected return type of the function call
- * @param functionName The name of the method to call.
- * @param arguments An optional list of expressions to pass as arguments to the function.
- * @param propagateNull When true, and any of the arguments is null, null will be returned instead
- *                      of calling the function.
- * @param returnNullable When false, indicating the invoked method will always return
- *                       non-null value.
- */
+  * Invokes a static function, returning the result.  By default, any of the arguments being null
```

Comment:
Why we change the comment style? Looks not consistent with others.

Reply:
Those additional spaces shouldn't be there, I've fixed them.